tests/test_precovery_index.py

```diff
@@ -44,8 +44,8 @@ def test_precovery():
         )
         orbits_keplerian.append(orbit)
 
-    # Load observations from h5 file
-    observations_df = pd.read_hdf(TEST_OBSERVATION_FILE)
+    # Load observations from csv file
+    observations_df = pd.read_csv(TEST_OBSERVATION_FILE)
```

Comment:
This might be a bit excessive but do we want to test the behavior between h5-based indexing and csv-based indexing since we are maintaining both? If so, we could add a unit test that indexes the observations from h5 and csv and then does a comparison on the output directory. Likely overkill so please don't feel like this needs to happen for this to be merged.

Reply:
While this PR maintains both, I would prefer to keep it simple and only go with the one file format. If we did maintain both, then yes I would definitely add a test.